mesh/mesh.go

```diff
@@ -109,19 +112,25 @@ func NewRecoveredMesh(db *MeshDB, atxDb AtxDB, rewardConfig Config, mesh MeshVal
 	}
 	ll.latestLayer = types.LayerID(util.BytesToUint64(latest))
 
-	validated, err := db.general.Get(VALIDATED)
+	processed, err := db.general.Get(PROCESSED)
 	if err != nil {
-		logger.Panic("could not recover validated layer: %v", err)
+		logger.Panic("could not recover processed layer: %v", err)
 	}
-	ll.validatedLayer = types.LayerID(util.BytesToUint64(validated))
+	ll.processedLayer = types.LayerID(util.BytesToUint64(processed))
 
 	if ll.layerHash, err = db.general.Get(LAYERHASH); err != nil {
 		logger.With().Error("could not recover latest layer hash", log.Err(err))
 	}
 
+	verified, err := db.general.Get(PBASE)
```

Comment:
basically pbase this is saved (and persisted) in the tortoise, so this may cause inconsistencies, i suggest pulling it out from the MeshValidator at startup.

Reply:
It is not exactly pbase, note that the latest layer pushed to state is `pbase - 1`, aslo note that crash can happen when pushing layers to state and may not even reach pbase -1 before crashing, this is why an extra field is needed, I can rename if you think the name is misleading